hw4.rb

```diff
@@ -0,0 +1,28 @@
+puts -123.odd?
+puts 6893.next
+puts 0.pred
+puts (12.to_f / 4.5).round
+puts ((28%6)*4.7).floor
+puts b1="Brand new bicycle\r\n".gsub('\r','').gsub('\n','')
+puts b2="Brand new bicycle".split('').delete_at(1)
```

Comment:
This will return the `r` character. Try splitting with whitespace (' '), or try using indexes to extract the word